src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,98 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int STATISTIC_POSITIONS_COUNT = 6;
+    static final int SUPPLY_POSITION_NUMBER = 0;
+    static final int BUY_POSITION_NUMBER = 2;
+    static final int RESULT_POSITION_NUMBER = 4;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = getDataFromFile(fromFileName);
+        String[]statistic = getStatisticPattern();
+        StringBuilder stringBuilder = new StringBuilder();
+        int sum;
+        for (String datum : data) {
+            String[] note = datum.split(",");
+            sum = Integer.parseInt(note[1]);
+            switch (note[0]) {
```

Comment:
lets remove these switches. all we need to do is read a line, split it by separator and calculate `buy` and `supply`